examples/00-systemcoupling/turek_hron_fsi2.py

```diff
@@ -0,0 +1,462 @@
+force_transfer.relaxation_factor = 0.5
+
+
+# %%
+# Time step size, end time, output controls
+syc.setup.solution_control.time_step_size = "0.01 [s]"  # time step is 0.01 [s]
+
+# To generate the results shown in the documents increase
+# this parameter to 20.0 s.
+syc.setup.solution_control.end_time = "0.1 [s]"  # end time
+
+syc.setup.output_control.option = "StepInterval"
+syc.setup.output_control.output_frequency = 250
+# print(syc.setup.get_setup_summary())
+
+# %%
+# Solve the coupled system
+# ------------------------
+syc.solution.solve()
+
+
+# %%
+# Postprocessing
+# ---------------
+
+# %%
+# Postprocess the structural results
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+mapdl.finish()
+mapdl.post1()
+mapdl.nsel("s", "loc", "x", 0.45)  # select the right side of the beam
+mapdl.nsel("r", "loc", "y", 0.00)  # select the top of the beam
+tip_node = mapdl.nsel("r", "loc", "z", 0.005)[0]  # select the tip node
+tip_y_0 = mapdl.get_value("node", tip_node, "loc", "y")
+tip_y = []
+nsets = mapdl.post_processing.nsets
+for i in range(1, nsets + 1):
+    mapdl.set(i, 1)
+    u_y = mapdl.get_value("node", tip_node, "u", "y")
+    tip_y.append(tip_y_0 + u_y)
+
+time_values = mapdl.post_processing.time_values
+plt.plot(time_values, tip_y)
+plt.xlabel("t (s)")
+plt.ylabel(r"$x_{tip}$ (m)")
+plt.savefig("turek_horn_fsi2_tip_disp.png")
+# If you want to see the in-line plot, comment following line.
+plt.close()  # close the plot to avoid showing it in the docs.
+
+###############################################################################
+# .. image:: /_static/turek_horn_fsi2_tip_disp.png
+#   :width: 500pt
+#   :align: center
+
+
+# %%
+# Postprocess the fluids results
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# use_window_resolution option not active inside containers or Ansys Lab environment
+
+if fluent.results.graphics.picture.use_window_resolution.is_active():
+    fluent.results.graphics.picture.use_window_resolution = False
+
+fluent.results.graphics.picture.x_resolution = 1920
+fluent.results.graphics.picture.y_resolution = 1440
+
+fluent.results.graphics.contour["contour_static_pressure"] = {}
+contour = fluent.results.graphics.contour["contour_static_pressure"]
+
+contour.coloring.option = "banded"
+contour.field = "pressure"
+contour.filled = True
+
+contour.surfaces_list = ["symmetry_bot"]
+contour.display()
+
+fluent.results.graphics.views.restore_view(view_name="front")
+fluent.results.graphics.views.auto_scale()
+fluent.results.graphics.picture.save_picture(
+    file_name="turek_horn_fsi2_pressure_contour.png"
+)
+
+###############################################################################
+# .. image:: /_static/turek_horn_fsi2_pressure_contour.png
+#   :width: 500pt
+#   :align: center
+
+
+# %%
+# Exit
+# ----
+
+syc.exit()
+fluent.exit()
+mapdl.exit()
+
+# %%
+# Note
+# ----
+# The results shown in this example are for illustrative purposes only.
+# To get accurate results, you may need to run the simulation for a till 15 [sec].
```

Comment:
There seems to be a typo in the sentence above ("... for a till 15 [sec]"). I am not sure exactly what it is meant to say.

Reply:
@iboyd-ansys 

Will update it to make it clear. 

Thanks!